core/nodejsActionBase/src/service.js

```diff
@@ -122,8 +185,10 @@ function NodeActionService(config) {
      *
      * req.body = { value: Object, meta { activationId : int } }
      */
-    this.runCode = function runCode(req) {
-        if (status === Status.ready && userCodeRunner !== undefined) {
+    this.runCode = runCode
+
+    function runCode(name, req) {
```

Comment:
This is used by both `/run` and `/web/api/v2/:namespace/:package/:action`. We can reuse `/run` and extract the actionName, package and namespace from the request body:
```bash
__OW_ACTION_NAME # /whisk.system/user274378/sampleJavaScript
__OW_NAMESPACE # /whisk.system
```

The request body contains the object `value` - which represents the input arguments, and any other object set in the payload besides `value` is set as ENV VAR for the action.